### PR TITLE
packge and deploy lab support for Cloud9

### PR DESCRIPTION
**Pull request by AvinashSeelam9**
## What does this PR do and why?
This PR is to provide Cloud9 support for package and deploy lab

_Describe in detail what your merge request does and why._
This PR is to provide Cloud9 support for package and deploy lab
## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by cjhar — COMMENTED** [review]
LGTM but no **Clean Up** section leaving a cfn stack and S3 bucket

File: content/intermediate/templates/package-and-deploy/index.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ From within a `code/workspace/package-and-deploy` directory run:
 :::code{language=shell showLineNumbers=false showCopyAction=true}
 aws cloudformation deploy \
     --template-file infrastructure-packaged.template \
-    --stack-name cfn-workshop-lambda \
+    --stack-name cfn-workshop-package-deploy-lambda \
     --region eu-west-1 \
```
Comment (cjhar) [inline content/intermediate/templates/package-and-deploy/index.md:225]:
```suggestion
    --region us-east-1 \
```

File: content/intermediate/templates/package-and-deploy/index.md [hunk 2]
```diff
@@ -301,6 +301,40 @@ more response.json
...
+    ::::
+    ::::tab{id="LocalDevelopment" label="LocalDevelopment"}
+      1. Navigate to the [AWS S3 Console](https://s3.console.aws.amazon.com/s3/).
+      1. Select the S3 bucket that you have created during this lab, Choose **Empty**.
+      1. Follow the instructions on the console to confirm the deletion of objects in the buckets.
+      1. Now switch back to S3 Console, Select the S3 bucket that you have created during this lab, Choose **Delete**
+      1. Follow the instructions on the console to confirm the deletion of the S3 bucket.
+      1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
+      1. Select the stack named `cfn-workshop-package-deploy-lambda` and choose **Delete**.
+      1. In the pop-up window click on Delete stack.
```
Comment (cjhar) [inline content/intermediate/templates/package-and-deploy/index.md:333]:
```suggestion
      1. In the pop-up window choose **Delete**.
```

**Review by cjhar — CHANGES_REQUESTED** [review]
Thanks only a couple of small fixes, plus the below that I couldn't suggest here
line 74 The `aws cloudformation package` performs the following actions:
line 87 aws s3 mb s3://example-bucket-name --region us-east-1
line 155 We know that `package` will generate ZIP files, so even there is no `.zip` extension you can still `unzip` it.
line 201 :::code{language=json showLineNumbers=false showCopyAction=false}

**Review by cjhar — DISMISSED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**